Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift

```diff
@@ -0,0 +1,216 @@
+import Flow
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct MovingFlowApartmentView: View {
+    @StateObject var vm = MovingFlowViewModel()
+
+    var body: some View {
+        hForm {}
+            .hFormTitle(.standard, .title1, L10n.changeAddressEnterNewAddressTitle)
+            .sectionContainerStyle(.transparent)
+            .hFormAttachToBottom {
+                hSection {
+                    VStack(spacing: 16) {
+                        VStack(spacing: 4) {
```

Comment:
When clicking on a field I have no way to dismiss the keyboard which covers the "send in" button :/